assemblyline_v4_service/common/result.py

```diff
@@ -266,6 +275,9 @@ def __init__(self) -> None:
         super().__init__(BODY_FORMAT.URL, body=[])
 
     def add_url(self, url: str, name: Optional[str] = None) -> None:
+        if not url:
```

Comment:
Should passing in an empty uri trigger a ValueError exception instead of simply returning silently?